src/cc65/declare.c

```diff
@@ -1773,9 +1773,14 @@ static void DefineData (ExprDesc* Expr)
 {
     switch (ED_GetLoc (Expr)) {
 
+        case E_LOC_NONE:
+            /* Immediate numeric value with no storage */
+            g_defdata (CF_IMM | TypeOf (Expr->Type) | CF_CONST, Expr->IVal, 0);
+            break;
+
         case E_LOC_ABS:
-            /* Absolute: numeric address or const */
-            g_defdata (TypeOf (Expr->Type) | CF_CONST, Expr->IVal, 0);
+            /* Absolute numeric address */
+            g_defdata (CF_ABSOLUTE | TypeOf(Expr->Type) | CF_CONST, Expr->IVal, 0);
```

Comment:
Please add a space after `TypeOf`.

Reply:
Added in d869e3a3b7050be0393dae7d93bcbd5652397c53.